test/tools/jsontests/vm.cpp

```diff
@@ -477,30 +477,11 @@ class VmTestFixture
 			cnote << "Skipping " << casename << " because --all option is not specified.\n";
 			return;
 		}
-		fillAllFilesInFolder(casename);
-	}
-
-	void fillAllFilesInFolder(string const& _folder)
-	{
-		using path = boost::filesystem::path;
-		path const fillersPath = path(test::getTestPath()) / "src/VMTestsFiller" / path(_folder);
-		string const filter = test::Options::get().singleTestName.empty() ? string() : test::Options::get().singleTestName + "Filler";
-
-		std::vector<boost::filesystem::path> const files = test::getJsonFiles(fillersPath.string(), filter);
-		size_t fileCount = files.size();
-		if (test::Options::get().filltests)
-			fileCount *= 2; //tests are checked when filled and after they been filled
-
-		auto testOutput = dev::test::TestOutputHelper(fileCount);
-		for (auto const& file: files)
-		{
-			test::TestOutputHelper::setCurrentTestFileName(file.filename().string());
-			test::executeTests(file.filename().string(), "/VMTests/"+_folder, "/VMTestsFiller/"+_folder, dev::test::doVMTests);
-		}
+		test::runAllTestsInFolder(test::testType::VMTests, casename);
 	}
 };
 
-
+std::string const test::c_VMTests = "VMTests";
```

Comment:
Why wouldn't you put this into `TestHelper.cpp` ? It's kind of hard to find this definition when declaration is in `TestHelper.h` and definition in some unpredictable file elsewhere

Reply:
because those strings should match the name of the boost test cases

Reply:
@gumb0 @winsvega the fundamental weird thing: the definitions in `vm.cpp` implement some declarations in `TestHelper.h`.  `doVMTests()` is declared in `TestHelper.h` and is defined in `vm.cpp`.